apps/web/src/data/ecosystem.json

```diff
@@ -3055,13 +3055,13 @@
     "imageUrl": "/images/partners/everyworld.png"
   },
   {
-    "name": "Circle.tech",
+    "name": "Time.fun",
     "tags": [
       "social"
     ],
-    "description": "Paywall and monetize your time by providing gated access and content to your most valuable fans.",
-    "url": "https://circle.tech",
-    "imageUrl": "/images/partners/circletech.jpeg"
+    "description": "Tokenize your time and allow your fans to get 1-on-1 time with you through unique experiences.",
+    "url": "https://time.fun",
+    "imageUrl": "/images/partners/timefun.png"
```

Comment:
@lucacurranCB 

Looks like this logo still needs to be added to `apps/web/public/images/partners`

Reply:
Did this in the wrong order. Image request is in the next PR (linked below)

https://github.com/base-org/web/pull/623